features/user.feature

```diff
@@ -152,6 +152,19 @@ Feature: Manage WordPress users
     Then STDERR should not be empty
     And the return code should be 1
 
+  Scenario: Trying to delete a existing user with no roles from subsite
+    Given a WP multisite install
+
+    When I run `wp user create bobjones bob@example.com --role=author --porcelain`
+    And save STDOUT as {BOB_ID}
+    And I try `wp user delete bobjones` 
```

Comment:
Can we move this into its own step with assertion?

```
When I run `wp user create bobjones bob@example.com --role=author --porcelain`
And save STDOUT as {BOB_ID}

When I run `wp user delete bobjones`
Then STDOUT should be
...

When I try `wp user delete bobjones`
Then STDERR should be...
```

Reply:
Yes, that works!